src/bakers-dozen-app/bakers-dozen-app.html

```diff
@@ -0,0 +1,31 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<dom-module id="bakers-dozen-app">
```

Comment:
This component, `<bakers-dozen-app>`, will be the highest-level component that holds everything about our game.